src/fssrgen/FsSrGen.fsproj

```diff
@@ -1,60 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I had originally been trying to make a netstandard 1.6 version of fssrgen but I kept running into issues with the xml related classes, so I decided to hold off on that and get the other stuff working first.